content/de/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html

```diff
@@ -9,7 +9,7 @@
 
 <body>
 
-<script src="https://katacoda.com/embed.js"></script>
+{{< kat-button >}}
```

Comment:
This is only present to trigger the removal notice; see `layouts/docs/single.html`